VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from '../logging/logger.service';
+
+import { IHearingControlsState, IHearingControlStates, VideoControlCacheService } from './video-control-cache.service';
+
+describe('VideoControlCacheService', () => {
+    let sut: VideoControlCacheService;
+    let loggerSpy: jasmine.SpyObj<LoggerService>;
+
+    beforeEach(() => {
+        loggerSpy = jasmine.createSpyObj<LoggerService>('LoggerService', ['warn', 'info']);
+        sut = new VideoControlCacheService(loggerSpy);
+
+        window.localStorage.clear();
```

Comment:
Should we mock localstorage for these tests?

Reply:
I was going to add a service for it but didn't see much of a benefit as these sort of tests will still exist just a layer down. I can add it though.

Reply:
I am going to refactor the video control service as there are some changes in the future which will mean that this will need doing